tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -353,43 +353,46 @@ public:
    /////////////////////////////////////////////////////////////////////////////
    /// \brief Create an RNTupleProcessor for a *join* (i.e., a horizontal combination) of RNTuples.
    ///
-   /// \param[in] ntuples A list specifying the names and locations of the RNTuples to process. The first RNTuple in the
-   /// list will be considered the primary RNTuple and drives the processor iteration loop. Subsequent RNTuples are
-   /// considered auxiliary, whose entries to be read are determined by the primary RNTuple (and do not necessarily
-   /// have to be in sequential order).
+   /// \param[in] primaryNTuple The name and location of the primary RNTuple, which will drive the processor iteration
+   /// loop.
+   /// \param[in] auxNTuples The names and locations of the RNTuples to join the primary RNTuple with. Which entries of
+   /// these auxiliary RNTuples are read is determined by the primary RNTuple, which does not necessarily happens in
```

Comment:
There is a small typo, but I actually wanted to ask a clarification regarding this sentence. What does not necessarily happen in sequential order?
```suggestion
   /// these auxiliary RNTuples are read is determined by the primary RNTuple, which does not necessarily happen in
```